src/Sunny.jl

```diff
@@ -89,6 +89,16 @@ export DynamicStructureFactor, InstantStructureFactor, StructureFactor, FormFact
     add_sample!, intensities, instant_intensities, broaden_energy, lorentzian,
     connected_path, all_exact_wave_vectors, ωs, spherical_shell
 
+include("SpinWave/SWUtil.jl")
+include("SpinWave/SpinWave.jl")
+export SpinWaveFields, construct_magnetic_supercell,
+    set_dipoles!, lswt_dispersion_relation, lswt_dynamical_spin_structure_factor!, lswt_unpolarized_INS_spec
+
+# include("WangLandau/BinnedArray.jl")
+# include("WangLandau/WangLandau.jl")
+# export BinnedArray, filter_visited, reset!,
+#     WangLandau, spherical_cap_update, init_bounded!, run!
```

Comment:
You probably noticed WangLandau/ got deleted -- these lines are no longer necessary, please remove.

Reply:
I have removed these lines.

Reply:
Thanks a lot. On this topic, maybe we should replace the name `polarize_spins!` with something more symmetric with `set_coherent_state!`